projects/core/src/cart/store/effects/cart.effect.ts

```diff
@@ -212,17 +199,8 @@ export class CartEffects {
   refresh$: Observable<
```

Comment:
It would be nice to reference it with a ticket number `TODO(#7241)` in a code comment